scripts/submodule-diff.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+git submodule foreach 'git describe HEAD --tags' > pre.log
```

Comment:
maybe do `| tee pre.log` here and below to ease debugging?

Reply:
Also for `post.log` then?